evaluator.py

```diff
@@ -55,7 +55,7 @@ def run(self, data):
                 positions: a list-like of dict-like objects.
         """
         dates = [item.date for item in data]
-        pnls = np.array([item.pnl for item in data])
+        pnls = np.array([sum(item.pnl.value()) for item in data])
```

Comment:
I think it is better to be
`pnls = {instrument: np.array(pnl) for instrument, pnl in item.pnl.items()}`
and modify how we calculate returns, sharpe, etc.
Otherwise there are no reasons why we should change the format -- we just wipe all new information immediately.

Reply:
But I think the change for how we calculate returns, sharpe, etc could be in next PR ?
Do this simple change (sum(item.pnl.values( )) ) could made PR for simulation merge into master.
And the change for return could be done simultaneously while doing other adjustments on other projects.

Reply:
isn't it `pnl in data.pnl.items()}` here?

Reply:
maybe I can leave the massage in the title to remind us, if we decide to leave the update of calculation later.

Reply:
Well, making changes in returns and sharpe is not that difficult actually.
If it will be a huge change, I would say it is reasonable.
But modification in both functions are just 1~2 lines of change.
For example, we can simply add `aggregated_pnls = sum(p.array(list(pnls.values())))` in both functions to apply for new format.

Reply:
@hcshen916 Yes it should be `data.pnl.items()`

Reply:
Yeah, we can do that and make it more beautiful in the future XD.

Reply:
The best solution here (to me) is to add a new column -- `PNLS_NP`, and generate it here from pnls.
In the case, all indicators can choose to use the np version if they don't really care what instruments are.

Reply:
This will not be a big change as well I think...